logstash-filter-geoip.gemspec

```diff
@@ -22,8 +23,9 @@ Gem::Specification.new do |s|
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", "~> 1.0"
 
-  s.add_runtime_dependency 'geoip', ['>= 1.3.2']
-  s.add_runtime_dependency 'lru_redux', "~> 1.1.0"
+  s.requirements << "jar com.maxmind.geoip2:geoip2, 2.5.0, :exclusions=> [com.google.http-client:google-http-client]"
+
+  s.add_development_dependency "jar-dependencies"
```

Comment:
you could also add the following so jar-dependencies doesn't wrangle it in when you do vendor_jars

    s.add_development_dependency 'ruby-maven', '~> 3.3'